src/postgres/Dockerfile

```diff
@@ -0,0 +1,8 @@
+# Copyright The OpenTelemetry Authors
```

Comment:
Created this image to avoid having to have a volume in Helm, just for the init script.